includes/admin/class-coblocks-crop-settings.php

```diff
@@ -86,26 +86,22 @@ public function hide_cropped_from_library( $query ) {
 	 * Retrieve the original image.
 	 */
 	public function get_original_image() {
-		$nonce = filter_input( INPUT_POST, 'nonce' );
-
-		if ( ! $nonce ) {
-
-			wp_send_json_error( 'No nonce value present.' );
-
+		if ( ! wp_verify_nonce( filter_input( INPUT_POST, 'nonce' ), 'cropSettingsOriginalImageNonce' ) ) {
```

Comment:
I'm wondering if we should wrap this filter_input with a `sanitize_text_field()` just to ensure it's sanitized before checking. This has been getting flagged in plugin reviews, and the security team might flag it if we don't.

```php
if ( ! wp_verify_nonce( sanitize_text_field( filter_input( INPUT_POST, 'nonce' ) ), 'cropSettingsOriginalImageNonce' ) ) {
```

Reply:
That's odd, considering it's not stored in a database or rendered on the website. We can add it regardless.

Reply:
Yeah, I agree. It throws a warning in our plugin scanner tool and in the plugin check plugin.